server.go

```diff
@@ -594,7 +594,9 @@ func statusFromError(p ider, err error) sshFxpStatusPacket {
 		ret.StatusError.Code = translateErrno(e)
 	case *os.PathError:
 		debug("statusFromError,pathError: error is %T %#v", e.Err, e.Err)
-		if errno, ok := e.Err.(syscall.Errno); ok {
+		if os.IsNotExist(err) {
```

Comment:
Hm… we might be able to move this up, before the type switch, with an early return.

```
…
ret.StatusErr.msg = err.Error()

if os.IsNotExist(err) {
  ret.StatusError.Code = sshFxNoSuchFile
  return ret
}

switch e := err.(type) {
…
```

Then we would also cover the below `default: switch { case os.ErrNotExist: … }` case in a more full and proper way as well.

Reply:
done.  I went ahead and simplified the final switch statement since it now only has a single case.